tools/Wasm.mk

```diff
@@ -24,7 +24,17 @@ include $(APPDIR)$(DELIM)interpreters$(DELIM)wamr$(DELIM)Toolchain.defs
 # wasi-sdk toolchain setting
 
 WCC ?= $(WASI_SDK_PATH)/bin/clang
-WAR ?= $(WASI_SDK_PATH)/bin/llvm-ar rcs
+WAR ?= $(WASI_SDK_PATH)/bin/llvm-ar
```

Comment:
but should we disable wasm build if nobody enables wasm related config in the first place?

Reply:
Here is the first place, Wasm.mk is introduced by Application.mk.

I think check it by toolchain is better, there is a case that enable a wasm interpreter but don't want to build wasm app from nuttx repo (some apps enabled WASM_BUILD by default).

Reply:
so should we enable wasm build until some app want to output wasm module?